Sources_v3/StreamChat/Database/DTOs/MessageDTO.swift

```diff
@@ -355,7 +374,7 @@ private extension _ChatMessage {
             isSentByCurrentUser = false
         }
         
-        attachments = Set(dto.attachments.map { $0.asModel() })
+        attachments = dto.attachments.map { $0.asModel() }
```

Comment:
We need to make sure the order here is stable.

Reply:
fixed in 0461a294116799fa8fb9b1fd30c2eec851a45490